docs: update readme docs (#5740)
PreMiD/Presences: README.md

<div align="center">
    <img src="https://cdn.rcd.gg/PreMiD-logo-text.svg" width="500px" style="max-width:100%;">
    <h1>PreMiD Presences</h1>
</div>

> This repository contains the source code of all presences that are available in PreMiD Store. 

Reviewer: English
```suggestion
This repository contains the source code of all the presences that are available in the [PreMiD Store](http://premid.app/store). 
```

Author: ririxi moment

This repository contains the source code of all presences that are available in PreMiD Store. 

<div align="left">
    <a target="_blank" href="https://discord.premid.app/" title="Join our Discord!">
        <img  src="https://discordapp.com/api/guilds/493130730549805057/widget.png?style=banner2" height="76px" draggable="false" alt="Join our Discord!">
    </a>
</div>

## Requirements to create a presence

- You must have the [latest TypeScript compiler](https://www.typescriptlang.org/#download-links) version installed
- Basic knowlege of TypeScript

## Getting started

We made a documentation to make your development process a little bit better and faster. You can see it [**here**](https://docs.premid.app/dev/presence).

## Folder structure

```bash
presence
├── dist
│   └── metadata.json
├── presence.ts
└── tsconfig.json
```

For presences with `iframe.ts`:

```bash
presence
├── dist
│   └── metadata.json
├── presence.ts
├── iframe.ts
└── tsconfig.json
```

---
## Documentation

> ## Translator Tool Guide

Reviewer: I agree the translator tool guide shouldn't be on the readme, but it should definitely be on CONTRIBUTING.md. You should also reference it below when you link that file

Author: No, it should be on PMD when that releases. It is a inactive tool anyway

Reviewer: Wdym inactive tool? Anyone can use this at any time, there has to be documentation

> We made a documentation to make your development process a little bit better and faster. You can read it [here](https://docs.premid.app/dev/presence).

Reviewer: seriously?
```suggestion
We have a documentation website to make the development process a little bit easier and faster. You can check it out it [here](https://docs.premid.app/dev/presence).
```

> If you have a question or a problem, reach out to us by joining our [Discord](https://discord.premid.app) and ask your question in [#presence-dev](https://discord.com/channels/493130730549805057/607524579874832446)

Reviewer: Do we really need a link to #presence-dev tho?
```suggestion
If you have a question or a problem, reach out to us by joining our [Discord server](https://discord.premid.app) and ask your question in [#presence-dev](https://discord.com/channels/493130730549805057/607524579874832446)
```
